diff --git a/src/unit_test/CMakeLists.txt b/src/unit_test/CMakeLists.txt
index 2010f145..661470fc 100644
--- a/src/unit_test/CMakeLists.txt
+++ b/src/unit_test/CMakeLists.txt
@@ -23,6 +23,11 @@ include_directories(${Dakota_INCLUDE_DIRS}/src) # include tpl directories
 # Needed when linking with QUESO
 link_directories(${Dakota_LIBRARY_DIRS})
 
+target_link_libraries(min_unit_test "rt")
+target_link_libraries(auto_unit_test "rt")
+target_link_libraries(bootstrap_util_test "rt")
+target_link_libraries(test_field_covariance_utils "rt")
+
 # ---
 # Unit test targets
 # ---
