diff --git a/packages/surfpack/src/CMakeLists.txt b/packages/surfpack/src/CMakeLists.txt
index fe3cd72f..d9eb40ac 100644
--- a/packages/surfpack/src/CMakeLists.txt
+++ b/packages/surfpack/src/CMakeLists.txt
@@ -72,6 +72,24 @@ add_library(${local_library} ${lib${local_library}_sources})
 add_library(${local_library}_fortran ${lib${local_library}_sources_fortran})
 target_link_libraries(${local_library} ${local_library}_fortran)
 
+option(SURFPACK_PYTHONBINDINGS "create python-bindings for surfpack" ON)
+
+if(SURFPACK_PYTHONBINDINGS)
+ 
+    EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} -c
+      "from distutils.sysconfig import get_python_lib; import os; print(get_python_lib())"
+      OUTPUT_VARIABLE _PYTHON_INSTALL_DIR OUTPUT_STRIP_TRAILING_WHITESPACE )
+
+    string(REPLACE "\\" "/" PYTHON_INSTALL_DIR ${_PYTHON_INSTALL_DIR})
+
+    pybind11_add_module(pysurfpack SHARED pysurfpack.cpp)
+    target_link_libraries(pysurfpack PRIVATE ${local_library} ${Pecos_LINK_LIBRARIES}
+      ${Teuchos_LIBRARIES} ${TEUCHOS_LIBRARY} ${LAPACK_LIBS} ${BLAS_LIBS} ${Boost_LIBRARIES}
+      ${SURFPACK_TPL_LIBS})
+
+    install(TARGETS pysurfpack DESTINATION ${PYTHON_INSTALL_DIR})
+endif()
+
 if(HAVE_BOOST_SERIALIZATION)
   target_compile_definitions(${local_library} PUBLIC
     SURFPACK_HAVE_BOOST_SERIALIZATION)
