diff --git a/interfaces/Python/CMakeLists.txt b/interfaces/Python/CMakeLists.txt
index 9d03dbfac..e93ae212c 100644
--- a/interfaces/Python/CMakeLists.txt
+++ b/interfaces/Python/CMakeLists.txt
@@ -8,7 +8,7 @@ if(DAKOTA_ENABLE_TESTS AND Python_Interpreter_FOUND AND
   set_property(TEST dakota_interfacing_test PROPERTY LABELS Unit Python)
 endif()
 
-install(DIRECTORY "dakota" DESTINATION "share/dakota/Python/"
+install(DIRECTORY "dakota" DESTINATION ${Python_SITEARCH}
   PATTERN "__pycache__" EXCLUDE
   PATTERN "*.py[cdo]" EXCLUDE
   )
